pkgs/overrides/oscavmgr.nix

```diff
@@ -0,0 +1,33 @@
+# SPDX-FileCopyrightText: 2025 Red_M <nixpkgs-xr@red-m.net>
+#
+# SPDX-License-Identifier: MIT
+
+final: prev: {
+  oscavmgr = prev.oscavmgr.overrideAttrs (prevAttrs: {
+    inherit (final.xrSources.oscavmgr)
+      pname
+      version
+      src
+      date
+      ;
+
+    cargoDeps = final.rustPlatform.importCargoLock final.xrSources.oscavmgr.cargoLock."Cargo.lock";
+
+    buildNoDefaultFeatures = true;
+    buildFeatures = [
+      "openxr"
+      "babble"
+    ]; # ALVR has a nasty bug (which doesn't swear, but its pretty aggressive), nixpkgs won't pull submodules down in cargo deps....
```

Comment:
I think I have already fixed this in my derivation in Nixpkgs: https://github.com/NixOS/nixpkgs/blob/5461b7fa65f3ca74cef60be837fd559a8918eaa0/pkgs/by-name/os/oscavmgr/package.nix#L39-L46